QueryBuilder/Helpers/ConditionHelper.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Did we really need to make a helper class for this? All of the other private/internal methods for creating specific conditions have lived inside the FilteredQuery.cs file

Reply:
I still feel like this is unnecessary because you can just create a constructor on WhereIsOfModelCondition that takes these parameters.

Reply:
Makes sense, will change.